gradle.properties

```diff
@@ -0,0 +1 @@
+excludeIntegrationTests=netezza,sqlserver,teradata
```

Comment:
I think we shouldn't have this `gradle.properties` here but rather in the test vm provisioning.